app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
     <string name="settings_report_issue">Report an issue</string>
+    <!-- LOCALIZATION NOTE (settings_version_developer): FIXME This string probably doesn't need to be localized/ -->
     <string name="settings_version_developer">Developer Build</string>
+    <!-- LOCALIZATION NOTE (settings_developer): The string 'settings_developer' is used to label a
+         button in the settings dialog that when pressed opens a dialog box that contains setting
+         an application or web developer might want to change. -->
     <string name="settings_developer">Developer</string>
+    <!-- LOCALIZATION NOTE (settings_options): FIXME this string is part of 'settings_developer' and is broken up to make it fit. -->
     <string name="settings_options">Options</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_restart): The string 'restart_dialog_restart' is the title of
+         a dialog box shown to the user when a settings change requires the application to restart. -->
     <string name="restart_dialog_restart">Restart Required</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_text): The string 'restart_dialog_text' is the text displayed
+         in the restart dialog box explaining why the restart is required. -->
     <string name="restart_dialog_text">You must restart the Firefox Reality app in order to complete changes. Would you like to do that now?</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_button):  The string 'restart_dialog_button' is used to
+         label the button the user presses to restart the application in the restart dialog. -->
     <string name="restart_dialog_button">Restart Now</string>
+    <!-- LOCALIZATION NOTE (developer_options_remote_debugging): The string 'developer_options_remote_debugging'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle remote debugging
+         of web content in the application. -->
     <string name="developer_options_remote_debugging">Enable Remote Debugging</string>
+    <!-- LOCALIZATION NOTE (developer_options_show_console): The string 'developer_options_show_console'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle redirecting
+         console output to the Android logcat. -->
     <string name="developer_options_show_console">Redirect Console to Logcat</string>
+    <!-- LOCALIZATION NOTE (developer_options_env_override): The string 'developer_options_env_override'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle loading externally
+         generated environments and overriding the ones built in. -->
     <string name="developer_options_env_override">Enable Environment Override</string>
+    <!-- LOCALIZATION NOTE (developer_options_multiprocess): The string 'developer_options_multiprocess'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle running content
```

Comment:
if I wasn't vaguely aware of how browsers work, I might not understand what this means: `running content in a separate process`

Reply:
These are unfortunately developer options so their function is rather technical. I'll see if I can add more context.